app/src/gui/layout/appBar.tsx

```diff
@@ -19,8 +19,8 @@
  *   throughout the app.
  */
 
-import React, {useEffect, useState} from 'react';
-import {Link as RouterLink, NavLink} from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { Link as RouterLink, NavLink } from 'react-router-dom';
 import {
```

Comment:
You'll need to merge in the most recent changes and run lint to get everything aligned with the gts code style.